ModularTegustation/tegu_items/associations/items.dm

```diff
@@ -67,13 +67,21 @@
 	to_chat(user, "<span class='nicegreen'>You suddenly feel different.</span>")
 	//Guarantee one
 	user.adjust_all_attribute_levels(amount)
+	to_chat(user, "<span class='nicegreen'>You gain 1 potential!</span>")
```

Comment:
This should definitely be a variable that's added up and a message is sent *once*. Also these should be using the new span wrappers.